src/core/issues-notes.js

```diff
@@ -118,7 +118,37 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (report.title && githubAPI) {
+          var labelListSpan = document.createElement('span');
+          Array.from(ghIssue.labels)
+            .forEach((label, index) => {
+              var labelSpan = document.createElement('span');
+              labelSpan.classList.add("respec-gh-label");
+              labelSpan.style.backgroundColor = `#${label.color}`;
+              labelSpan.style.textTransform = 'none';
+              labelSpan.id = `${dataNum}-label-${index}`;
+              labelSpan.innerText = label.name;
+              const color = label.color.substring(1);
```

Comment:
All the color code should move to it’s own “deriveTextColorClass()” that returns “white” or “black”.

Reply:
Actually, it should return “light” or “dark”.

Reply:
Yep got it. I will update this in a while.